static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+          "summary": "Delete policy",
+          "tags": [
+            "Policies"
+          ],
+          "security": [
+            {
+              "ApiKeyAuth": []
+            }
+          ],
+          "description": "Delete policy by ID.",
+          "parameters": [
+            {
+              "in": "path",
+              "name": "id",
+              "schema": {
+                "type": "string",
+                "example": "87315f9f-0ef5-5499-ac6b-086ff488e3dd"
+              },
+              "required": true,
+              "description": "ID of the policy to be deleted."
+            }
+          ],
+          "operationId": "deletePolicy",
+          "responses": {
+            "204": {
+              "description": "No content."
+            },
+            "400": {
+              "description": "Bad Request."
+            },
+            "401": {
+              "description": "Authorization failure due to invalid authentication credentials."
+            },
+            "403": {
+              "description": "Either the API key doesn't have permissions or it is deactivated."
+            },
+            "404": {
+              "description": "Policy ID not found.",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "type": "object",
+                    "properties": {
+                      "error": {
+                        "type": "boolean",
+                        "example": true
+                      },
+                      "type": {
+                        "type": "string",
+                        "example": "P3"
+                      },
+                      "errorMessage": {
+                        "type": "string",
+                        "example": "Policy ID not found."
+                      },
+                      "code": {
+                        "type": "number",
+                        "example": 404
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "500": {
+              "description": "Internal server error."
+            }
+          }
+        }
+      },
+      "/api/p3/v1/validations/validateHtml/ondemand": {
+        "post": {
+          "summary": "Validate HTML on-demand",
+          "tags": [
+            "Validations"
+          ],
+          "description": "Validates the HTML of a given URL and returns a detailed visual and functional comparison result.",
+          "operationId": "validateHtmlOnDemand",
+          "requestBody": {
+            "required": true,
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "url": {
+                      "type": "string",
+                      "format": "uri",
+                      "example": "https://www.samsung.com/us/shop/all-deals/tv-and-home-theater/"
```

Comment:
Stay away from using specific complies names in examples.  Use something like www.example.com.